### DISCUSSION TITLE
Is it possible to remove abstract & file from being exported to .bib file?

### DISCUSSION THREAD

**Discussion by DS-Rodrigues** [retorquere/zotero-better-bibtex#2465] [Q&A] [answered]
Hi

Is it possible to remove the following 2 fields from being exported to a .bib file via Better BibTex:

- abstract
- file

Many thanks,
Daniela

**Comment by retorquere** [comment, marked as answer]
Yes: https://retorque.re/zotero-better-bibtex/installation/preferences/export/#fields-to-omit-from-export-comma-separated

**Reply by DS-Rodrigues** [reply]
This is great, thanks!
